src/utils/useMousePosition.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```

Comment:
This was ripped from https://www.joshwcomeau.com/snippets/react-hooks/use-mouse-position/, may use something different before this PR gets merged, just wanted to iterate with something like this.